Readme.md

```diff
@@ -108,36 +112,45 @@ optional arguments:
 ```
 
 
-If you would like, you can install HistoQC into your system by using 
+
+Alternatively one can specify their own modified config file using an absolute or relative filename:
 
 ```
-git clone https://github.com/choosehappy/HistoQC.git
-cd HistoQC
-python -m pip install --upgrade pip  # (optional) upgrade pip to newest version
-pip install -r requirements.txt      # install pinned versions of packages
-pip install .
+python -m histoqc.config --show light > mylight.ini
+python -m histoqc -c ./mylight.ini -n 3 "*.svs"
 ```
 
-Installed or simply git-cloned, a typical command line for running the tool thus looks like:
+## histoqc.ui CLI
+
+HistoQC now has a httpd server which allows for improved result viewing, it can be accessed like so:
 
 ```
-python -m histoqc -c v2.1 -n 3 "*.svs"
-```
+C:\Research\code\HistoQC>python -m histoqc.ui --help
+usage: __main__.py [-h] [--bind ADDRESS] [--port PORT] [--deploy OUT_DIR]
+                   [data_directory]
 
-which will use 3 process to operate on all svs files using the named configuration file config_v2.1.ini from the config directory.
+positional arguments:
+  data_directory        Specify the data directory [default:current directory]
 
-Alternatively one can specify their own modified config file using an absolute or relative filename:
+optional arguments:
+  -h, --help            show this help message and exit
+  --bind ADDRESS, -b ADDRESS
+                        Specify alternate bind address [default: all
+                        interfaces]
+  --port PORT           Specify alternate port [default: 8000]
+  --deploy OUT_DIR      Write UI to OUT_DIR
 
 ```
-python -m histoqc.config --show light > mylight.ini
-python -m histoqc -c ./mylight.ini -n 3 "*.svs"
-```
 
+After completion of slide processing, view results in your web-browser simply by running the following command *from within the output directory* (saved in the **histoqc_output_YYMMDD-hhmmss** format by default. See histoqc CLI -o option)
 
-Afterward completion of slide processing you can view the results in your web-browser simply by following the directions after typing:
+```
+python -m histoqc.ui 
```

Comment:
i'd probably just add a single line here:

cd histoqc_output_YYMMDD-hhmmss
python -m histoqc.ui

to further really reinforce that you're supposed to be in *that* sub-directory. i find people often don't read blocks of text but read code, so this'll hint it at them in case they missed it

this does assume however that histoqc was pip installed if they're in the sub directory, right? otherwise i don't think python -m works?

might want to add that as a comment, and then on the next line where you have the output directory specified say "OR set data_d...... (or if histoqc was not installed)"

or something like that?

Reply:
Sure. Since your last point applies to all modules of histoqc, I added a note in the installation section.